data/overlays/gamestates/z_file_nameset_PAL.rodata.s

```diff
@@ -0,0 +1,156 @@
+.include "macro.inc"
+
+ # assembler directives
+ .set noat      # allow manual use of $at
+ .set noreorder # don't insert nops after branches
+ .set gp=64     # allow use of 64-bit general purposee registers
+
+.section .rodata
+
```

Comment:
All sections should start with `.balign 16` to properly align them to file boundaries.
```suggestion

.balign 16

```